crates/language/src/csharp.rs

```diff
@@ -63,11 +62,25 @@ impl Language for CSharp {
         &[("", "")]
     }
 
-    fn metavariable_sort(&self) -> SortId {
-        self.metavariable_sort
+    fn is_comment(&self, node: &NodeWithSource) -> bool {
+        MarzanoLanguage::is_comment_node(self, node)
+    }
+
+    fn is_metavariable(&self, node: &NodeWithSource) -> bool {
+        MarzanoLanguage::is_metavariable_node(self, node)
+    }
```

Comment:
These are unfortunately reoccurring. Code that is otherwise TreeSitter-agnostic wants to know whether something is a comment or a metavariable, but the implementations for that are tied to TreeSitter, so we end up forwarding from one trait to the other.